python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_condition.py

```diff
@@ -193,6 +193,33 @@ def eager() -> "SchedulingCondition":
             & ~SchedulingCondition.in_progress()
         )
 
+    @staticmethod
+    def on_cron(cron_schedule: str, cron_timezone: str = "UTC") -> "SchedulingCondition":
+        """Returns a condition which will materialize asset partitions within the latest time window
+        on a given cron schedule, after their parents have been updated.
+
+        Specifically, this is a composite SchedulingCondition which is true for an asset partition
+        if all of the following are true:
+
+        - The asset partition is within the latest time window
+        - All parent asset partitions have been updated since the latest tick of the provided cron
+            schedule, or will be requested this tick
+        - The asset partition has not been updated since the latest tick of the provided cron schedule
+        - The asset partition is not currently part of an in-progress run
+        """
```

Comment:
Is this how we would do the basic cron job? Meaning the one where you can annotate the root or all assets in a group with this and it just works like a cron?

We should have at least one test for this, even if all it does it call the static constructor and very it returns an instance of SchedulingCondition to catch some silly error and get code coverage.

Reply:
Yep exactly, this would replace a basic cron job